aws-ssm-maintenancewindowtarget/src/main/java/software/amazon/ssm/maintenancewindowtarget/util/ResourceModelToStringConverter.java

```diff
@@ -0,0 +1,51 @@
+package software.amazon.ssm.maintenancewindowtarget.util;
+
+import software.amazon.ssm.maintenancewindowtarget.ResourceModel;
+
+/**
+ * Used for safe printing of {@link ResourceModel} objects.
+ */
+public class ResourceModelToStringConverter {
+
+    /**
+     * Converts ResourceModel objects into Strings that are safe to store. No confidential data will be returned.
+     *
+     * @param resourceModel Model object that needs to be converted into String.
+     * @return String representation of the model object that is safe to store.
+     */
+    public String convert(final ResourceModel resourceModel) {
+        if (resourceModel == null) {
+            return "null";
+        }
+
+        return String.format("ResourceModel(" +
+                "clientToken=%s," +
+                "description=%s," +
+                "filters=%s," +
+                "maxResults=%s," +
+                "name=%s," +
+                "nextToken=%s," +
+                "ownerInformation=%s," +
```

Comment:
We probably shouldn't log OwnerInformation, Name, Description as they are sensitive data. I'm not sure if we need to log replace and safe. It seems like replace is set to true by default?